docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,157 @@
+# Each WDS consists of a mandatory unique name and several optional parameters:
+# - type: host|k8s (default to k8s, if not specified)
+# - APIGroups: a comma separated list of APIGroups
+# - ITSName: the name of the ITS control plane to be used by the WDS. Note that the ITSName MUST be specified if more than one ITS exists.
+WDSes: # ==> installs kubestellar + ocm-transport-plugin
+```
+
+The first section of the `values.yaml` file refers to parameters that are specific to the KubeFlex instllation, see [here](https://github.com/kubestellar/kubeflex/blob/main/docs/users.md) for more information.
+
+In particular:
+- `kubeflex-operator.install` accepts a boolean value to enable/disable the installation of KubeFlex into the cluster by the chart
+- `kubeflex-operator.isOpenShift` must be set to true by the user when installing the chart into a OpenShift cluster
+
+By default, the chart will install the KubeFlex and its PostgreSQL dependency.
+
+The second section allows a user of the chart to determine if Post Create Hooks (PCHs) needed for creating ITSes and WDSes control planes should be instlalled by the chart. By default `InstallPCHs` is set to `true` to enable the instllation of the PCHs, however one mya want to set this value to `false` when installing multiple copies of the chart to avoid conflicts. A single copy of the PCHs is required and allowed per cluster.
+
+The third section of the `values.yaml` file allows one to create a list of Inventory and Transport Spaces (ITSes). By default, this list is empty and no control plane will be created by the chart. A list of ITSes can be specified using the following format:
+
+```yaml
+ITSes: # ==> installs ocm + ocm-status-addon
+  - name: <its1>          # mandatory name of the control plane
+    type: <vlcuster|host> # optional type of control plane host|vcluster (default to vcluster, if not specified)
+  - name: <its2>          # mandatory name of the control plane
+    type: <vlcuster|host> # optional type of control plane host|vcluster (default to vcluster, if not specified)
+  ...
+```
+
+where `name` must specify a unique name of the control plane and the optional `type` can be either vlcuster (default) or host, see [here](https://github.com/kubestellar/kubeflex/blob/main/docs/users.md) for more information.
+
+The fourth section of the `values.yaml` file allows one to create a list of Workload Description Spaces (WDSes). By default, this list is empty and no control plane will be created by the chart. A list of WDSes can be specified using the following format:
+
+```yaml
+WDSes: # ==> installs kubestellar + ocm-transport-plugin
+  - name: <wds1>     # mandatory name of the control plane
+    type: <host|k8s> # optional type of control plane host|k8s (default to k8s, if not specified)
+    APIGroups: ""    # optional list of APIGroups
+    ITSName: <its1>  # optional name of the ITS control plane, this MUST be specified if more than one ITS exists

```

Comment:
More than one ITS exists _when_? Since usage of this chart changes the number of ITSes, the reader needs to be told which point in time matters here.

Reply:
"at the moment the WDS PCH is executed"

Reply:
Execution of a PCH takes place over time. Perhaps you mean when the PCH _starts_ ?

Reply:
ok